Adding a Project Task from Project Module in Gantt Chart view hangs/fails

<!--- Provide a general summary of the issue in the **Title** above -->
<!--- Before you open an issue, please check if a similar issue already exists or has been closed before. --->

#### Issue
The Project Gantt Chart view contains a button to add a new task to the project. When clicked it opens up a "Add New Task" modal. When the "add" button is clicked nothing appears to happen on the front end though web server cpu usage spikes significantly for a short period. 

When a user clicks "Cancel" in the Add New Task popup it displays a "Loading Please Wait" message with a broken icon image and closes the window.

#### Expected Behavior
When the "add" button is clicked the new task should be created and added to the project and the add task modal should close. The Gantt chart should be updated either via Ajax if enabled or by page refresh. 

#### Actual Behavior
Nothing happens when "Add" is clicked. No task is created or added to the project and the Add Task window remains open. 

#### Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->

#### Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug include code to reproduce, if relevant -->
1. Create a Project or navigate to an existing one. 
2.  In the Project view click the Actions dropdown and click View Gantt
3. Click the "+ Add New Task" button and input the required fields for the task.
4. Click "Add"

Error log snippet created:
`Tue Feb 14 11:17:01 2017 [2942][id][FATAL] error loading relationship quotes
Tue Feb 14 11:17:40 2017 [32263][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:17:40 2017 [32263][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:17:40 2017 [32263][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:17:40 2017 [32263][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:19:40 2017 [32266][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:19:40 2017 [32266][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:19:40 2017 [32266][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:19:40 2017 [32266][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:19:40 2017 [32266][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:19:40 2017 [32266][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:45:05 2017 [3761][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:45:05 2017 [3761][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:45:05 2017 [3761][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:45:05 2017 [3761][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:52:05 2017 [3253][id][ERROR] Unable to find relationship projects_quotes
Tue Feb 14 11:52:05 2017 [3253][id][FATAL] quotes for projects_quotes failed to load

Tue Feb 14 11:52:05 2017 [3253][id][ERROR] Unable to find relationship projects_quotes
Tue Feb 14 11:52:05 2017 [3253][id][FATAL] quotes for projects_quotes failed to load

Tue Feb 14 11:52:10 2017 [3784][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:52:10 2017 [3784][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:52:10 2017 [3784][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 11:52:10 2017 [3784][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 12:07:05 2017 [4127][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 12:07:05 2017 [4127][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 12:07:05 2017 [4127][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 12:07:05 2017 [4127][id][ERROR] Unable to get proper side for link project_name_link
Tue Feb 14 12:07:32 2017 [4122][id][FATAL]  Query Failed: DESCRIBE versions: MySQL error 1146: Table 'crm.versions' doesn't exist
Tue Feb 14 12:07:32 2017 [4122][id][FATAL]  Query Failed: SHOW INDEX FROM versions: MySQL error 1146: Table 'crm.versions' doesn't exist
Tue Feb 14 12:07:32 2017 [4122][id][FATAL]  Query Failed: select * from versions: MySQL error 1146: Table 'crm.versions' doesn't exist`

#### Context
<!--- How has this bug affected you? What were you trying to accomplish? -->
<!--- If you feel this should be a low/medium/high priority then please state so -->
I would view this as a medium priority issue. Tasks can still be added via alternative means however this is a functionality issue that blatantly does not work for the end user.
#### Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* SuiteCRM Version used:  7.8.0 Sugar Version 6.5.24 (Build 509) any theme.
* Browser name and version: Chrome Version 56.0.2924.87
* Environment name and version (e.g. MySQL, PHP 7): PHP 7.0, MySQL  5.0.12
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 16.04.1

--- a/modules/Project/controller.php
+++ b/modules/Project/controller.php
@@ -134,7 +134,7 @@ class ProjectController extends SugarController
 
             if ($bh) {
                 $bh = $bh[0];
-                if ($bh->open) {
+                if ($bh->open_status) {
                     $open_h = $bh ? $bh->opening_hours : 9;
                     $close_h = $bh ? $bh->closing_hours : 17;
 

